i2c/26-i2c.html

```diff
@@ -58,7 +58,7 @@
     </div>
    <div class="form-row">
         <label for="node-input-count" alt="How many bytes will be returned"><i class="fa fa-sign-in"></i> Bytes</label>
-        <input type="number" id="node-input-count" class="I2C-in-count" placeholder="1" min=0 max=31>
+        <input type="number" id="node-input-count" class="I2C-in-count" placeholder="[msg.bytes]" min=0 max=31>
```

Comment:
why is that a better place holder than a number ?  maybe "number or msg.bytes" would be better ?

Reply:
i think that msg.bytes is a better placeholder (but "number or msg.bytes" is perfect too, maybe even better) because if a user deletes the default "1" value he can see that it is possible to directly pass the desired value through msg.bytes

Reply:
OK - make it "number or msg.bytes"